bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/RRD4jPersistenceService.java

```diff
@@ -191,24 +191,31 @@ public void store(final Item item, @Nullable final String alias) {
 
         long now = System.currentTimeMillis() / 1000;
 
-        scheduler.schedule(() -> internalStore(name, value, now, true), 0, TimeUnit.SECONDS);
+        scheduler.schedule(() -> internalStore(name, value, null), 0, TimeUnit.SECONDS);
```

Comment:
Please remove the line just before that set variable `now` that is no more used.